packages/webapp/src/components/Map/ProgressBar/index.jsx

```diff
@@ -37,7 +41,7 @@ export default function ProgressBar({ onDismiss, type = 'success' }) {
     return () => {
       clearInterval(timer);
     };
-  }, []);
+  }, [onDismiss]);
 
   useEffect(() => {
     if (progress >= MAX_PROGRESS) {
```

Comment:
This component was always receiving `onDismiss` prop, so `onDismiss?.()` (on the next line) was always being called.
Now, `onDismiss` is conditionally passed from the parent component.